.moban.yaml

```diff
@@ -28,8 +28,9 @@ entry_points:
     - coala-quickstart = coala_quickstart.coala_quickstart:main
 
 dependencies:
-  - coala_bears~=0.12.0.dev20170722110839
-  - coala_utils~=0.6.6
+  # PR 5706, 7806, and 5809
+  - https://github.com/coala/coala/archive/e4e780de.zip
```

Comment:
iirc according to the syntax it should be `/archive/branch_name.zip` right? what is `e4e780de`? 😅

Reply:
you can look this up in pep documentation.